Makefile

```diff
@@ -331,14 +331,20 @@ importer-image: PLATFORMS=linux/amd64
 importer-image: PUSH=--load
 importer-image: importer-image-build
 
+# Build multiarch kueue-viz image locally
+.PHONY: local-kueue-viz-image-build
+local-kueue-viz-image-build:
+	BUILDER=$(shell $(DOCKER_BUILDX_CMD) create --use)
+	$(MAKE) kueue-viz-image-build PUSH=$(PUSH)
+	$(DOCKER_BUILDX_CMD) rm $$BUILDER
 
 # Build the kueue-viz dashboard images (frontend and backend)
 .PHONY: kueue-viz-image-build
 kueue-viz-image-build:
 	$(IMAGE_BUILD_CMD) \
 		-t $(IMAGE_REGISTRY)/kueue-viz-backend:$(GIT_TAG) \
 		-t $(IMAGE_REGISTRY)/kueue-viz-backend:$(RELEASE_BRANCH)-latest \
-		--platform=$(VIZ_PLATFORMS) \
+		--platform=$(PLATFORMS) \
```

Comment:
Why we changed it?

Reply:
The VIZ_PLATFORMS was added since it wasn't working with multiarch builds. I figured it would be safe to remove and prefer the more generic `PLATFORMS` environment variable

Reply:
Ah, I see, they are the same. In that case, could you please remove VIZ_PLATFORMS everywhere?